cypress/integration/pages/testsForAllAMPPages.js

```diff
@@ -1,9 +1,31 @@
-const testsForAllAMPPages = pageType => {
+import envConfig from '../../support/config/envs';
+import config from '../../support/config/services';
+
+const serviceIsGNL = service => service === 'japanese';
+
+const testsForAllAMPPages = ({ service, pageType }) => {
   describe(`AMP Tests`, () => {
     if (pageType !== 'errorPage404') {
       it('should have an AMP attribute on the page', () => {
         cy.get('html').should('have.attr', 'amp');
       });
+      describe('ATI', () => {
+        it('should have an amp-analytics tag with the ati url', () => {
+          if (service === serviceIsGNL) {
+            cy.hasAmpAnalyticsAtiUrl(
+              envConfig.atiUrl,
+              envConfig.atiAnalyticsGNLBucket,
+            );
+          } else {
```

Comment:
It'd be clearer to read to have a single if/elseif/else and no ternary. e.g.

 if (service === serviceIsGNL) {
...
} else if (config[service].isWorldService) {
...
} else {
...
}